packages/react/src/isomorphicClerk.ts

```diff
@@ -1151,7 +1151,7 @@ export class IsomorphicClerk implements IsomorphicLoadedClerk {
     }
   };
 
-  authenticateWithOKXWallet = async (params: AuthenticateWithOKXWalletParams): Promise<void> => {
+  authenticateWithOKXWallet = async (params?: AuthenticateWithOKXWalletParams): Promise<void> => {
```

Comment:
Typedoc complained about type errors here, so I fixed those. It's weird that it doesn't happen for other functions above that also have optional `params`